src/getty-generator/getty-generator.c

```diff
@@ -147,8 +157,20 @@ static int add_credential_gettys(void) {
                                 break;
                         }
 
-                        if (startswith(tty, "#"))
+                        if (startswith(tty, "#")) {
+                                const char *comment = skip_leading_chars(tty, WHITESPACE);
+
+                                if (streq(comment, "no-auto"))
```

Comment:
Oh, I did not notice these are parsed from the credential files...
Hm, we should not mix the purpose of the credential files like that.

How about to introduce `systemd.getty_mode=none|auto|(comma separated list of 'credentials', 'container', 'console', and 'builtins')` kernel command line option and `getty.mode` credentials that can specify the same value?